File: src/java/org/codehaus/testdox/intellij/actions/ToggleQuickDoxAction.java
package org.codehaus.testdox.intellij.actions;

import com.intellij.openapi.actionSystem.AnActionEvent;
import org.codehaus.testdox.intellij.IconHelper;

public class ToggleQuickDoxAction extends BaseAction {

    public ToggleQuickDoxAction() {
        super("Toggle QuickDox", "Shows/hides the TestDox tooltip", IconHelper.getIcon(IconHelper.DOX_ICON));
    }

    public void actionPerformed(AnActionEvent event) {
        actionEvents.getTestDoxController(event).toggleQuickDox();
    }
}

File: src/scala/org/codehaus/testdox/intellij/actions/ToggleQuickDoxAction.scala
package org.codehaus.testdox.intellij.actions

import com.intellij.openapi.actionSystem.AnActionEvent
import org.codehaus.testdox.intellij.IconHelper._
import org.codehaus.testdox.intellij.IconHelper

class ToggleQuickDoxAction extends BaseAction("Show QuickDox", "Shows TestDox in a tooltip", getIcon(IconHelper.DOX_ICON)) {

  def actionPerformed(event: AnActionEvent) {
    actionEvents.getTestDoxController(event).toggleQuickDox()
  }
}
